### PR TITLE
Custom recordId zod type to Schema

### PR DESCRIPTION
**Pull request by CiskaLV**
Just added the custom recordID zod type to the Schemas and shortened the _generated import path

### CODE REVIEW

File: src/genSchema/generateTableSchema.ts [hunk 1]
```diff
@@ -96,7 +96,8 @@ export ${outputFields};
 
 import { z } from "zod";
 
-import { ${tableName}InputSchemaGen, ${tableName}OutputSchemaGen } from "../../_generated/${tableName}/${tableName}SchemaGen.js";
+import { ${tableName}InputSchemaGen, ${tableName}OutputSchemaGen } from "../../_generated";
```
Comment (sebastianwessel) [inline src/genSchema/generateTableSchema.ts:99]: It must include at least `index.js`, because of ESM

File: src/genSchema/generateTableSchema.ts [hunk 2]
```diff
@@ -106,7 +107,7 @@ export const ${tableName}CreateSchema = ${tableName}InputSchemaGen.merge(z.objec
 
 // payload schema for fetching a ${name} entity
 export const ${tableName}Schema = ${tableName}OutputSchemaGen.merge(z.object({
-  id: z.object({ tb: z.string(), id: z.string() }),
+  id: recordId("${tableName}"),
```
Comment (sebastianwessel) [inline src/genSchema/generateTableSchema.ts:110]: 👍🏼

**Review by sebastianwessel — CHANGES_REQUESTED** [review]

**Comment by CiskaLV** [comment]
I added index.js and also added an optional id to the CreateSchema so if you want to make a record with a set id you can

File: src/genSchema/generateTableSchema.ts (hunk 1 above)
Comment (CiskaLV) [reply src/genSchema/generateTableSchema.ts]: Fixed this

**Review by sebastianwessel — APPROVED** [review]

**Pull request merged**